data/json/vehicle_parts.json

```diff
@@ -251,10 +251,19 @@
         "broken_symbol" : "#",
         "broken_color" : "brown",
         "durability" : 100,
+        "requirements": {
+          "install": {
+            "skills": [ [ "mechanics", 0 ] ],
```

Comment:
As stated in comment, was emulating hardcoded behavior of `NAILABLE` excepting the change of component. Alternative skills would be just fine if you prefer I do so.